.circleci/config.yml

```diff
@@ -89,28 +89,28 @@ workflows:
             - build_go_images
             - build_dashboard
             - build_pinniped_proxy
-      - GKE_1_15_MASTER:
+      - GKE_1_17_MASTER:
           <<: *build_on_master
```

Comment:
can you temporarily change this to `build_always` to run the tests in these platforms? (also you will need to push this branch to the kubeapps repo, as explained offline)

Reply:
Yes, of course. In fact, I did it after creating two "PR-able" branches, you can find the CI results in: https://app.circleci.com/pipelines/github/kubeapps/kubeapps?branch=test-changes-ci

Reply:
ah, I see, IMO is easier for you to to actually make the PR from that branch because if you need to modify something you can just do it there (rather than having to migrate the changes here). Also it's easier to link the tests executed to the PR.

Reply:
Mmm, make sense... but, since the change actually requires two PRs (update the GKE env and fix the positional params), I would have to perform this temporary change (+ revert it)  twice.
That's why I chose to merge both branches and add the build-always fix, but, yep, I agree it's easier to keep track of the test results.

Reply:
yeah, it's good to split changes in different PRs but it's not a law, you can do several things in a PR if there are not too many changes, more if they are related :P